src/CMakeLists.txt

```diff
@@ -39,6 +39,56 @@ endif (WIN32)
 # Include CREATE_DEBUG_SYM macro and set DEBUG_BUILD
 include (CreateDebugSym)
 
+# Find executable needed by gmt
+find_program (OGR2OGR ogr2ogr)
+if (NOT OGR2OGR)
+	message (WARNING "Cannot find program ogr2ogr - gmt will be unable convert Shapefiles to GMT/OGR format")
+	set (GMT_CONFIG_OGR2OGR_MESSAGE "no" CACHE INTERNAL "OGR2OGR config message")
+else (NOT OGR2OGR)
+	set (GMT_CONFIG_OGR2OGR_MESSAGE "yes" CACHE INTERNAL "OGR2OGR config message")
+endif (NOT OGR2OGR)
+# Find executables needed by psconvert
+find_program (GS NAMES gs gswin64c gswin32c)
```

Comment:
`find_program (GS NAMES gs gswin64c gswin32c)` doesn't work for Windows, as psconvert searches gswin32c/gswin64c by reading registry.

Reply:
So perhaps I just place that under if (UNIX)?

Reply:
The true reason is that GS doesn't add to the PATH so any program that wants to find it must look into the registry.

Reply:
So is this also true for ogr2ogr, ffmpeg, gm, etc., etc? I.e., place those under a UNIX test?

Reply:
find_program can search the paths via registry.
See https://github.com/Kitware/CMake/blob/master/Modules/FindCygwin.cmake.

Reply:
> So is this also true for ogr2ogr, ffmpeg, gm, etc., etc? I.e., place those under a UNIX test?

I don't think we should place all those under a UNIX test. For Windows, if cmake cannot find `ogr2ogr` in PATH, then we also won't be able to run it by a system call.

As I understand it, `gs` is the only special case, because we search it both in PATH and in registry.

Reply:
OK, I will only isolate the gs check.

Reply:
@seisman and @joa-quim, let me know if this is OK now.

Reply:
I agree it's good enough.

Reply:
OK, need someone to review and OK